index.js

```diff
@@ -93,6 +93,25 @@ app.put('/data/:id', async (req, res) => {
     }
 });
 
+app.delete('/data/:id', async (req, res) => {
+  try {
+      const { id } = req.params;
+      const result = await db.query(`
+        DELETE FROM test 
+        WHERE id =${id}
+        RETURNING * `);
+  
+      if (!result.rows[0]) {
+        return res.status(404).json({ error: "Resource not found" });
+      }
+  
+      res.json({data: result.rows[0] , mesagge: "Delete sucessfully"});
```

Comment:
Es mejor este mensaje 
```suggestion
      res.json({data: result.rows[0] , mesagge: "Data deleted successfully"});
```